cmd/start.go

```diff
@@ -181,11 +181,10 @@ func runContainer(cmd *cobra.Command, args []string) {
 			if !testEmptyDir {
 				log.Fatal(getUnderlyingStorage(flavor) + " is not empty, doing nothing.")
 			}
-			if runtime.GOOS == "linux" {
```

Comment:
Why did you remove this check? This Selinux thing is onlu valid on Linux.

Reply:
Added the check for linux environment .